packages/now-static-build/src/frameworks.ts

```diff
@@ -145,6 +168,7 @@ export const frameworks: Framework[] = [
     slug: 'angular',
     dependency: '@angular/cli',
     minNodeRange: '10.x',
```

Comment:
You can remove `minNodeRange` since it doesn't do anything anymore

Reply:
done 👍